### PR TITLE
Hacktoberfest writeup for the Ember Times

### PR DESCRIPTION
**Pull request by jenweber**
Here's a Hacktoberfest writeup! I borrowed from https://blog.emberjs.com/hacktoberfest

## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

### CODE REVIEW

File: content/the-ember-times-issue-190.md [hunk 1]
```diff
@@ -10,19 +10,46 @@ tags:
...
+### Ember projects you can help out with
+
+Are you wondering what to work on?
+
+There are [81 Ember projects participating](https://github.com/search?q=topic%3Aember+topic%3Ahacktoberfest)!
+Here's a list of [recently updated issues](https://github.com/search?o=desc&q=topic%3Aember+topic%3Ahacktoberfest&s=updated&type=Repositories) in those repositories.
+
+Ember 4.0 is coming up, and so one area of need is to help addon maintainers
+upgrade to version 3.28 and resolve deprecations. Even if addon authors
+do not have issues open for this work, you can open one and offer to help.
```
Comment (ijlee2) [inline content/the-ember-times-issue-190.md]:
```suggestion
Ember 4.0 is coming up, and so one area of need is to help addon maintainers
upgrade to version 3.28 and [resolve deprecations](https://deprecations.emberjs.com/). Even if addon authors
do not have issues open for this work, you can open one and offer to help.
```

I wonder if upgrading an addon with deprecated patterns to 3.28 is too big of a pull request. If a large pull request doesn't get reviewed and merged immediately, the contributor's effort won't be recognized on time for Hacktoberfest.

I think we could instead suggest opening an issue to remove a deprecation or refactor code, as a concrete, small step towards upgrading an addon.

**Review by ijlee2 — APPROVED** [review]
Looking forward to see new and returning contributors at Hacktoberfest!

File: content/the-ember-times-issue-190.md (hunk 1 above)
Comment (jenweber) [reply content/the-ember-times-issue-190.md]: Thank you Isaac, that’s an excellent point. I applied your suggestion.

**Comment by amyrlam** [comment]
thanks for doing this. i'll take a stab at grooming some of the issues on https://github.com/ember-learn/ember-blog/issues

File: content/the-ember-times-issue-190.md [hunk 2]
```diff
@@ -10,19 +10,46 @@ tags:
...
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+To participate:
+
+- Sign up anytime between October 1 and October 31.
+- Make 4 approved Pull Requests to public repositories on GitHub that have the `hacktoberfest` topic.
+Take a moment to learn about what goes into an ["approved" Pull request](https://hacktoberfest.digitalocean.com/resources/participation).
+- If you are among the first 50,000 developers who complete the challenge, you get a prize. This year, you can select between a T-shirt and a tree planted in your name.
+- Join the `#hacktoberfest` channel in the [Ember Community Discord](https://discord.gg/emberjs)
```
Comment (amyrlam) [inline content/the-ember-times-issue-190.md]:
```suggestion
- Join the [`#hacktoberfest`](https://discord.com/channels/480462759797063690/496453502298750988) channel in the [Ember Community Discord](https://discord.gg/emberjs)
```

**Comment by amyrlam** [comment]
lgtm

**Pull request merged**